pdc/apps/release/tests.py

```diff
@@ -1829,16 +1829,32 @@ def test_retrieve_with_name(self):
         response = self.client.get(reverse("releasegroups-detail", args=["rhel_test"]))
         expect_result = {'active': True, 'type': u'Async',
                          'name': u'rhel_test', 'releases': [u'release-1.0'],
-                         'description': u'good'}
+                         'description': u'test'}
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.assertEqual(response.data, expect_result)
 
+    def test_override_ordering_by_description_key(self):
+        response = self.client.get(reverse("releasegroups-list"), format='json')
+        expect_result = {'active': True, 'type': u'Async',
+                         'name': u'rhel_test', 'releases': [u'release-1.0'],
+                         'description': u'test'}
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+        self.assertEquals(response.data.get('results')[0], expect_result)
+        response1 = self.client.get(reverse("releasegroups-list"), {'ordering': 'description'},
+                                    format='json')
+        expect_result1 = {'active': True, 'type': u'QuarterlyUpdate',
+                          'name': u'rhel_test1', 'releases': [u'release-1.0'],
+                          'description': u'good'}
+        self.assertEqual(response1.status_code, status.HTTP_200_OK)
+        self.assertEquals(response1.data.get('results')[0], expect_result1)
+
```

Comment:
It's better to test multiple fields ordering and order with `-`. If we support use them together, it looks should be tested also.